### DISCUSSION TITLE
Is it able to detect who downloads a lot but reports little progress?

### DISCUSSION THREAD

**Discussion by ghost** [Simple-Tracker/qBittorrent-ClientBlocker#135] [General]
It seems there is no such a function (Sorry if I missed). I have found someone have actually downloaded a lot but continues reporting little progress, which looks like a PCDN pretending as a normal torrent client. Is it advisable to detect whether or not a peer reports normally increasing progress?

**Comment by Simple-Tracker** [comment]
Currently, the IP list provided by PBH-BTN can be used to solve this problem. (Default)
